compiler/testData/diagnostics/tests/namedArguments/disallowForJavaConstructor.fir.kt

```diff
@@ -6,4 +6,4 @@ public class A {
 
 // FILE: 1.kt
 
-val test = A(x = 1, y = "2")
+val test = A(<!NAMED_ARGUMENTS_NOT_ALLOWED!>x = 1<!>, y = "2")
```

Comment:
Why isn't the diagnostic on both `x` and `y`? It seems to be the case for FE1.0.

Reply:
Nice catch! Looks like the current infrastructure code needs to be changed to allow multiple reports.

Reply:
Doesn't it make sense to split the commit: one for checking NAMED_ARGUMENT_... and the other to extend the infra to allow multiple reports?